src/spring/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '1.13.1'
+VERSION = '1.13.2'
```

Comment:
I see that the PR #6358 also upgraded the extension version to `1.13.2`.

Reply:
it's two different features, can be released in diffrent version.

Reply:
As synced with @yilims offline, I have merged my change into his PR. Because we didn't explain the preview badge removal for GA features in our previous releases, I don't update the history in Yi's PR.